_posts/2020-05-02-schedule-your-functions.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Keep your functions running to schedule with the cron-connector.
+description: "In this post Martin will show you how to schedule your functions with the cron-connector, for regular tasks and scheduled jobs"
+date: 2020-05-02
+image: /images/2020-05-02-schedule-your-functions/accuracy-alarm-clock-analogue.jpg
+categories:
+  - cron
+  - kubernetes
+  - serverless
+  - linux
+  - tutorial
+  - example
+author_staff_member: martin
+dark_background: true
+
+---
+
+In this post Martin will show you how schedule your functions with the cron-connector, for regular tasks and scheduled jobs.
+
+One of the things we need very often is our functions to be executed on a schedule. This is where the cron-connector shines. We will showcase how you can use it in this post, making sure your functions always run on time.
```

Comment:
Mention in the opening about Lockbot and its GitHub repo

Reply:
Yep this is a good idea :+1:

Reply:
General idea mentioned :heavy_check_mark: